code/code/disc/disc_wrath.cc

```diff
@@ -15,103 +15,102 @@
 
 int plagueOfLocusts(TBeing *caster, TBeing *victim, int level, short bKnown)
 {
-  TMonster *locusts;
-  affectedData aff; 
-  int rc, swarm;
-  followData *k, *n;
-
   if (!caster->outside()) {
     act("Nothing seems to happen.", FALSE, caster, NULL, NULL, TO_ROOM);
     caster->sendTo("You need to be outside to summon locusts!\n\r");
     return SPELL_FAIL;
   }
+
   if (Weather::getWeather(*caster->roomp) == Weather::RAINY) {
     act("Nothing seems to happen.", FALSE, caster, NULL, NULL, TO_ROOM);
     caster->sendTo("How do you expect to summon locusts in the rain!\n\r");
     return SPELL_FAIL;
   }
+
+  if ((caster->followers) && (level < GOD_LEVEL1))  {
+    followData *currentFollower, *nextFollower;
+    for (currentFollower = caster->followers; currentFollower; currentFollower = nextFollower) {
+      nextFollower = currentFollower->next;
```

Comment:
Why do you need that variable anyway?

Reply:
Honestly not sure - I preserved the original logic and just renamed the variables to be more descriptive rather than single letters. I'll look at it further and try to simplify it.

Reply:
Fixed in https://github.com/sneezymud/sneezymud/pull/353/commits/e678035f48a6a98e660100b84771f378f1a8a648